quicly-probes.d

```diff
@@ -30,7 +48,7 @@ provider quicly {
                  struct st_quicly_address_token_plaintext_t *address_token);
     probe free(struct st_quicly_conn_t *conn, int64_t at);
     probe send(struct st_quicly_conn_t *conn, int64_t at, int state, const char *dcid);
-    probe receive(struct st_quicly_conn_t *conn, int64_t at, const char *dcid, const void *bytes, size_t bytes_len);
+    probe receive(struct st_quicly_conn_t *conn, int64_t at, const char *dcid, uint8_t first_octet, const void *bytes, size_t bytes_len);
```

Comment:
```suggestion
    probe receive(struct st_quicly_conn_t *conn, int64_t at, const char *dcid, uint8_t first_byte, const void *bytes, size_t bytes_len);
```

As stated above, `bytes` does not need to be marked as "appdata." Therefore I think it might be possible to leave this line unchanged, though I would not mind making a change, presuming that the new parameter would be named `first_byte` rather than octet. In the past, QUIC drafts were ambiguous in which term to use, but we concluded that "byte" should be used everywhere.

Reply:
That's good to know. 

I'll remove the change just because of its redundancy, but please let me know if you'd like to see `first_byte` in the output of h2olog.